docs/content/querying/sorting-orders.md

```diff
@@ -47,3 +47,6 @@ When comparing two unparseable values (e.g., "hello" and "world"), this ordering
 
 ## Strlen
 Sorts values by the their string lengths. When there is a tie, this comparator falls back to using the String compareTo method.
+
+## Version
+Sorts values as versions, e.g.: "10.0 sorts after 9.0"
```

Comment:
Are there any docs that could be linked to that describe the behavior of this sorting? It might help differentiate between this and `alphanumeric`, which also would sort this example correctly, but fails to order for example `1.0.0-SNAPSHOT` before `1.0.0`.

Reply:
Thanks so much, I'll add some docs.